Library/Homebrew/cmd/desc.rb

```diff
@@ -48,19 +52,32 @@ def desc
       :desc
     end
 
-    results = if search_type.nil?
+    if search_type.nil?
       desc = {}
-      args.named.to_formulae.each { |f| desc[f.full_name] = f.desc }
-      Descriptions.new(desc)
+      args.named.to_formulae_and_casks.each do |formula_or_cask|
+        if formula_or_cask.is_a? Formula
+          desc[formula_or_cask.full_name] = formula_or_cask.desc
+        else
+          description = formula_or_cask.desc.presence || Formatter.warning("[no description]")
+          desc[formula_or_cask.full_name] = "(#{formula_or_cask.name.join(", ")}) #{description}"
```

Comment:
Why are we displaying the results like `token: (name) description`? I know that `name` is a required field in a cask, but what does the `name` provide to users that `token` doesn't in this output? It feels a bit superfluous. Forgive my ignorance, I don't know much about casks!

Reply:
Well, that's what's up for debate. When displaying cask results with `desc` or `search`, should it tack the name onto the beginning of the description? Or the end? Or not at all, but still check its contents when searching? For most casks the name reflects the token, but in rare cases it has additional keywords, such as https://formulae.brew.sh/cask/smlnj or https://formulae.brew.sh/cask/igv or (an extreme example) https://github.com/Homebrew/homebrew-cask-drivers/blob/master/Casks/canon-ijscanner1.rb.